packages/termux-tools/login

```diff
@@ -1,13 +1,82 @@
-#!/data/data/com.termux/files/usr/bin/sh
+#!/system/bin/sh
+# ^ If the user compiles dash by hand with libedit, this will fail
+# if LD_LIBRARY_PATH is set.
```

Comment:
Typo here? "If LD_LIBRARY_PATH is not set"?

Reply:
Good catch.

I'm like a spell check. I can tell when a word is spelled wrong or whether I used "whom" properly, but I have trouble noticing when something is just plain wrong. :(

I'll fix it when I get home.